conf/ocsci/vault_external_standalone_mode.yaml

```diff
@@ -0,0 +1,12 @@
+DEPLOYMENT:
+  kms_deployment: true
+
+ENV_DATA:
+  encryption_at_rest: true
+  vault_deploy_mode: external
+  KMS_PROVIDER: vault
+  KMS_SERVICE_NAME: vault
+  VAULT_CACERT: "ocs-kms-ca-secret"
+  VAULT_CLIENT_CERT: "ocs-kms-client-cert"
+  VAULT_CLIENT_KEY: "ocs-kms-client-key"
+  VAULT_SKIP_VERIFY: false
```

Comment:
Don't you set `VAULT_BACKEND_PATH` here as well?

Reply:
@leseb Either user can pass the VAULT_BACKEND_PATH in yaml to override default else we will be generating backend path name something like "ocs-<CLUSTER-UUID>" per test run so that multiple runs can use same external vault instance.